service/src/main/java/com/codeforcommunity/processor/BlocksProcessorImpl.java

```diff
@@ -144,18 +144,20 @@ private GetAssignedBlocksResponse getAssignedBlocksWithStatus(BlockStatus status
             .fetchInto(AssignedBlock.class));
   }
 
+  /*
+   * Never use this in production. Uncomment and use locally, if needed. Don't commit the uncommented code.
+   * TODO: Create superuser privilege level for system management.
```

Comment:
don't use this except for local testing

Reply:
+1 on the `TODO`

Reply:
aha, PriviledgeLevel is broken because the enum is 1, 2 and the DB store is default 0. That'll have to be fixed separately  from this.

Reply:
I'm pretty sure it's just that the DB stores the enum value one lower... It's weird but it's not broken. Otherwise all of our previous calls would have failed